scripts/doc_checker.py

```diff
@@ -120,7 +120,9 @@ def check_optional_args(doc: Docstring) -> list:
 
 def check_spelling_words(doc: Docstring) -> list:
     """
-    Check spelling of chosen words: "Modin", "pandas", "NumPy", "Ray", "Dask" in doc.
+    Check spelling of chosen words in doc.
+
+    Words: "Modin", "pandas", "NumPy", "Ray", "Dask", "PyArrow", "OmniSci", "XGBoost".
```

Comment:
I don't think we need to specify the words here - it's unnecessary duplication IMHO, they're specified a few dozen lines below.